snp_motion_planning/src/planning_server.cpp

```diff
@@ -143,34 +160,10 @@ class PlanningServer
 
     // Create the environment monitor
     tesseract_monitor_ =
-        std::make_shared<tesseract_monitoring::EnvironmentMonitor>(node_, env_, TESSERACT_MONITOR_NAMESPACE);
-    tesseract_monitor_->startStateMonitor(tesseract_monitoring::DEFAULT_JOINT_STATES_TOPIC, false);
-
-    // Register custom process planners
-    planning_server_->registerProcessPlanner(TRANSITION_PLANNER, createTransitionTaskflow());
-    planning_server_->registerProcessPlanner(FREESPACE_PLANNER, createFreespaceTaskflow());
-    planning_server_->registerProcessPlanner(RASTER_PLANNER, createRasterTaskflow());
-
-    // Add custom profiles
-    {
-      auto pd = planning_server_->getProfiles();
-      pd->addProfile<tesseract_planning::SimplePlannerPlanProfile>(
-          tesseract_planning::profile_ns::SIMPLE_DEFAULT_NAMESPACE, PROFILE, createSimplePlannerProfile());
-      pd->addProfile<tesseract_planning::OMPLPlanProfile>(tesseract_planning::profile_ns::OMPL_DEFAULT_NAMESPACE,
-                                                          PROFILE, createOMPLProfile());
-      pd->addProfile<tesseract_planning::TrajOptPlanProfile>(tesseract_planning::profile_ns::TRAJOPT_DEFAULT_NAMESPACE,
-                                                             PROFILE, createTrajOptToolZFreePlanProfile());
-      pd->addProfile<tesseract_planning::TrajOptCompositeProfile>(
-          tesseract_planning::profile_ns::TRAJOPT_DEFAULT_NAMESPACE, PROFILE, createTrajOptProfile());
-      pd->addProfile<tesseract_planning::DescartesPlanProfile<float>>(
-          tesseract_planning::profile_ns::DESCARTES_DEFAULT_NAMESPACE, PROFILE, createDescartesPlanProfile<float>());
-      pd->addProfile<tesseract_planning::CheckInputProfile>(
-          tesseract_planning::profile_ns::CHECK_INPUT_DEFAULT_NAMESPACE, PROFILE,
-          std::make_shared<tesseract_planning::CheckInputProfile>());
-      pd->addProfile<tesseract_planning::SeedMinLengthProfile>(
-          tesseract_planning::profile_ns::SEED_MIN_LENGTH_DEFAULT_NAMESPACE, PROFILE,
-          std::make_shared<tesseract_planning::SeedMinLengthProfile>(5));
-    }
+        std::make_shared<tesseract_monitoring::ROSEnvironmentMonitor>(node_, env_, TESSERACT_MONITOR_NAMESPACE);
+    tesseract_monitor_->setEnvironmentPublishingFrequency(30.0);
+    tesseract_monitor_->startPublishingEnvironment();
+    tesseract_monitor_->startStateMonitor("/robot_joint_states", true);
```

Comment:
```suggestion
    tesseract_monitor_->startStateMonitor(tesseract_monitoring::DEFAULT_JOINT_STATES_TOPIC, false);
```
We should revert this change; we should listen to the full joint states topic for updates, and we shouldn't be publishing TF because we already have a robot state publisher running that is doing this. We can change in the future if we think we need to, but let's keep this as-is for now